LICENSE

```diff
@@ -564,4 +564,63 @@ ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
 CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 ========================================================================
+The libarchive distribution as a whole is Copyright by Tim Kientzle
```

Comment:
This needs to be reworked to follow the LICENSE and NOTICE guidance of the project found here: https://nifi.apache.org/licensing-guide.html.  The LICENSE file is exclusively for LICENSE language.  Some of the more ad-hoc written items could belong in the NOTICE but strictly only necessary information should belong there.